### DISCUSSION TITLE
Running only using terminal

### DISCUSSION THREAD

**Discussion by Tortoise17** [lllyasviel/Fooocus#2639] [Q&A] [answered]
Is there any way by flag or any file which makes it useable to generate the image without opening browser, only using the terminal with CLI.? I am trying to run it by using only terminal. Any idea would be great help.

**Comment by mashb1t** [comment, marked as answer]
This is only possible using a terminal based websockets client, but I've never done this before and wouldn't recommend to do so at all.
There is neither a native terminal only mode nor support for it.

**Comment by Tortoise17** [comment]
Thank you. But is it possible to start the client like this and enter the text by command line via terminal instead of browser? I mean same port works but the text is provided by terminal or any file manual input.

**Reply by mashb1t** [reply]
see answer above.
